src/topology/algebra/continuous_affine_map.lean

```diff
@@ -0,0 +1,147 @@
+/-
+Copyright (c) 2021 Oliver Nash. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Oliver Nash
+-/
+import linear_algebra.affine_space.affine_map
+import topology.metric_space.basic
+import topology.continuous_function.basic
+
+/-!
+# Continuous affine maps.
+
+This file defines a type of bundled continuous affine maps.
+
+## Main definitions:
+
+ * `continuous_affine_map`
+
+## Notation:
+
+We introduce the notation `P →A[R] Q` for `continuous_affine_map R P Q`. Note that this is parallel
+to the notation `E →L[R] F` for `continuous_linear_map R E F`.
+-/
+
+/-- A continuous map of affine spaces. -/
+structure continuous_affine_map (R : Type*) {V W : Type*} (P Q : Type*) [ring R]
+  [add_comm_group V] [module R V] [metric_space P] [add_torsor V P]
```

Comment:
Sorry I didn't catch this before -- can you please use `topological_space P` throughout this file rather than `metric_space P`?

Reply:
Already done ;-)